### PR TITLE
Change pwm msg

### PR DESCRIPTION
**Pull request by vegkams**
Move conversion from microseconds to bits to the PWM node.

### CODE REVIEW

File: vortex_msgs/msg/Pwm.msg [hunk 1]
```diff
@@ -1,3 +1,2 @@
 uint16[] pins
-uint16[] on
-uint16[] off
+uint16[] microseconds
```
Comment (sindrehan) [inline vortex_msgs/msg/Pwm.msg]: Rename to positive_width_us

**Review by sindrehan — CHANGES_REQUESTED** [review]

**Comment by sindrehan** [comment]
[Change pwm message to μs](https://trello.com/c/MinfIb4C/212-change-pwm-message-to-%CE%BCs)

File: vortex_msgs/msg/Pwm.msg [hunk 2]
```diff
@@ -1,2 +1,5 @@
+# PWM pin nrs
```
Comment (sindrehan) [inline vortex_msgs/msg/Pwm.msg]: > nrs

Really?

**Review by sindrehan — APPROVED** [review]
Seems good. Needs testing before merging.

File: vortex_msgs/msg/Pwm.msg (hunk 2 above)
Comment (vegkams) [reply vortex_msgs/msg/Pwm.msg]: Did you mean rly?

**Pull request merged**